### DISCUSSION TITLE
"Ignore All" as a Default Action for "The target name length (...) is more than 259 characters!"

### DISCUSSION THREAD

**Discussion by iMasry-Git** [doublecmd/doublecmd#1189] [General]
Hello,

My target is to have a copy/move process with least messages that pauses the process, and requires regular monitoring.

Is there an option to tell Doubled Commander to ignore the wanning about The target name length, and copy/move the files anyways?

In other words, to set the "Ignore All" as a Default Action for "The target name length (...) is more than 259 characters!" message?

![2023-08-04 16_29_34-Double Commander 1 0 11~272](https://github.com/doublecmd/doublecmd/assets/67241646/4d0bf788-eb85-45d4-952f-5ae5a7eee3b5)

**Comment by iMasry-Git** [comment]
If not, I see this can be added as a feature request.

**Comment by alexx2000** [comment]
Hi, see [LongNameAlert](https://doublecmd.github.io/doc/en/configxml.html#manualed) option.

**Reply by iMasry-Git** [reply]
Thanks! That’s it.
